src/ed.cpp

```diff
@@ -193,6 +142,7 @@ bool srvUpdate(ed_msgs::UpdateSrv::Request& req, ed_msgs::UpdateSrv::Response& r
                     if (!r.readValue("name", prop_name))
                         continue;
 
+                    // ToDo: is this thread safe?
```

Comment:
Shouldn't this get done then? same at line 202

Reply:
This is out of the scope. Of this PR. We haven't been experiencing any issues related these, but I added these comments for the future, because there are some concerns there.

Reply:
"concerns" is an under-statement I think :)